### DISCUSSION TITLE
再現性のない強制終了の報告

### DISCUSSION THREAD

**Discussion by Syumiru** [okuokuch/pokemon_sv_vs_support#184] [Bug Report]
### 概要
スプレッドシートとの接続完了後、パーティ読込の段階でPGMが強制終了しました。
### 内容詳細
再実行したところ、問題なく実行できました。
バグと断定し兼ねますが、一応報告させていただきます。
### 補足資料
[app.log](https://github.com/okuokuch/pokemon_sv_vs_support/files/12810104/app.log)

**Comment by okuokuch** [comment]
こちら原因の目星がついているので、共有します。
## 生じるタイミング
テンプレート画像は存在するのに、そのファイル名からポケモン名に変換するマスターデータにそのテンプレート画像名(+その変換先のポケモン名)のデータがない場合に発生。
DLPで新規追加されたポケモンに関して画像を追加したが、マスターデータ(poke_id.csv)の更新漏れがあるために発生。

## 今後の対応予定
poke_id.csvの更新。現在仮で更新したものがあるので一旦ver2ブランチに反映します。
一般の方向け公開は、使ってみてエラーが発生しないこと確認出来たら通知しようと思います。
